bin/information-gathering.sh

```diff
@@ -0,0 +1,259 @@
+save_file 32 proc-modules          /proc/modules      # Currently loaded kernel modules.
+save_file 33 proc-mounts           /proc/mounts       # List of all mounts in use by system.
+save_file 34 proc-partitions       /proc/partitions   # Detailed info about partitions available to the system.
+save_file 35 proc-pci              /proc/pci          # Information about every PCI device.
+save_file 36 proc-stat             /proc/stat         # Record or various statistics kept from last reboot.
+save_file 37 proc-swap             /proc/swap         # Information about swap space.
+save_file 38 proc-uptime           /proc/uptime       # Uptime information (in seconds).
+save_file 39 proc-version          /proc/version      # Kernel version, gcc version, and Linux distribution installed.
+
+# Save partition table
+save_command 40 mount              mount
+
+# Test DNS resolution
+save_command 41 dns                ping -c 3 google.com
+
+# Test ping/internet connectivy
+save_command 42 ping               ping -c 3 8.8.8.8
+
+# Check disk space usage
+save_command 43 disk-usage         df -h
+
+# Check the current language
+save_command 44 current-language   set | egrep '^(LANG|LC_)'
+
+# Save network information
+save_command 45 network-info       ifconfig -a
+
+# Save IP addresses
+save_command 46 ip-addresses       ip addr
+
+# Save network interfaces
+save_file    47 network-interfaces /etc/network/interfaces
+
+# Save routing information
+save_command 48 routing-info       route -n
+
+# Save all open ports
+save_command 49 open-ports         netstat -tulpn
+
+# Save who is logged in
+save_command 50 who-is-logged-in   who
+
+# Save list of all running processes
+save_command 51 running-processes  ps faux
+
+# Save current user
+save_command 52 current-user       id
+
+# Save current date
+save_command 53 current-date       date
+
+# Save current UTC date
+save_command 54 current-utc-date   date --utc
+
+# Save routing tables
+save_command 55 routing-tables     ip route
+
+# Save uptime
+save_command 56 uptime             uptime
+
+# Run ubuntu-report and copy the output file
+ubuntu-report --non-interactive 2>/dev/null
+cp -f ~/.cache/ubuntu-report/* $WORK_FOLDER/57-ubuntu-report.txt
+
+# Save timezone
+save_command 58 timezone           timedatectl
+
+# Save locale
+save_command 59 locale             locale
+
+# Save APT sources
+save_file 60 apt-sources           /etc/apt/sources.list
+
+# Save APT sources.list.d files
+save_command 61 apt-files          ls -l /etc/apt/sources.list.d/*
+
+# Save APT sources.list.d contents
+save_command 62 apt-contents       cat -n /etc/apt/sources.list.d/*
+
+# Save crontab
+save_file 63    crontab            /etc/crontab
+
+# Save Cron files
+save_command 64 cron-files         ls -l /etc/cron.d/*
+
+# Save Cron contents
+save_command 65 cron-contents      cat -n /etc/cron.d/*
+
+# Save DNSMasq configuration
+save_file 66    dnsmasq-conf       /etc/dnsmasq.conf
```

Comment:
our customers should never have a dnsmasq.conf.

Reply:
removed the step, shifted all numbers.